src/defs/clientPoliciesRepresentation.ts

```diff
@@ -0,0 +1,9 @@
+import ClientPolicyRepresentation from './clientPolicyRepresentation';
+
+/**
+ * https://www.keycloak.org/docs-api/15.0/rest-api/#_clientpoliciesrepresentation
+ */
+
+export default interface ClientPoliciesRepresentation {
+  policies: ClientPolicyRepresentation[];
```

Comment:
According to the documentation this is an optional field. So it should be marked as optional in the interface as well.
```suggestion
  policies?: ClientPolicyRepresentation[];
```